zapier/introduction/first-zap/zap-editor-2.md

```diff
@@ -0,0 +1,45 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Adding and Editing Zaps 2
+
+---
+## Content
+
+Setting up the Slack integration is similar to the Google Sheets set up. In fact, most of the integration will follow a similar process, with variations depending on the data or action type.
+
+Now, we want to choose `'Send Direct Message'` in the `'Choose Action Event'` drop-down and press `'Continue'`. 
+
+The next step is signing in to Slack. If you do this for the first time it might be a good idea to use a test workspace before integrating your production workspace.
+
+Note: Zapier adds a lot of permissions to your account, so if you don't feel comfortable you should check the entire list and decide if you want to go further or not. In the off chance that you do not, you can always choose another app to integrate.
+
+For this example, we want to receive a direct message every time someone edits the `"Has Access?"` column.
+
+In the next screen you will encounter a lot of fields that need to be completed. 
+
+For now, focus on completing the ones tagged as `'required'`, as these are the only ones needed for your zap to work:
+1. `'To Username'`
+    - the user that will receive the message
+    - for this example, we will give our own username
+2. `'Message Text'`
+    - the message that will be send on each update
+    - we will use `'A new person has gained access to your base!'`
+
+Don't worry about the other fields as they will be explained in the following insight.
+
+---
+## Practice
+
+In Zapier, you need to complete ??? to integrate your apps:
+
+* the required fields
+* the fields marked as 'optional'
```

Comment:
I decided to use 'this' because from what I remember, all the answers of a PQ or RQ are rendered `like this`. What I mean is that this:
```
* possible answer
```

is rendered like this:

`possible answer`

I'll test this in Spider and report back